security/s2a/internal/crypter/ciphersuite.go

```diff
@@ -0,0 +1,102 @@
+package crypter
+
+import (
+	"crypto/sha256"
+	"golang.org/x/crypto/sha3"
+	s2a_proto "google.golang.org/grpc/security/s2a/internal"
+	"hash"
+)
+
+type ciphersuite interface {
+	// keySize returns the key size in bytes.
+	keySize() int
+	// nonceSize returns the nonce size in bytes.
+	nonceSize() int
+	// expectedTrafficSecretSize returns the expected traffic secret size in
+	// bytes.
+	expectedTrafficSecretSize() int
```

Comment:
Can we call this `trafficSecretSize`? This would keep the naming in line with the other functions.

Reply:
I called it `expectedTrafficSecretSize` because that is what it is called in the C version. I thought there might be a reason why it had the `expected` prepended to it whereas the other variables don't. I think just `trafficSecretSize` is more fitting though, so I've changed it.